cpu/stm32_common/periph/dac.c

```diff
@@ -57,12 +88,8 @@ int8_t dac_init(dac_t line)
 void dac_set(dac_t line, uint16_t value)
 {
     value = (value >> 4);       /* scale to 12-bit */
-    if (dac_config[line].chan) {
-        DAC->DHR12R2 = value;
-    }
-    else {
-        DAC->DHR12R1 = value;
-    }
+
+    DAC_WRITE_OUTPUT(line, value);
```

Comment:
I would like to, but only the `STM32F0x1` family have a 1 channel DAC. And I don't want to clutter the code with defines like `#ifndef CPU_MODEL_STM32F051R8`